CHANGELOG.md

```diff
@@ -2,11 +2,47 @@
 
 ## Table of Contents
 
+- **[r1.2](#r12)**
 - [r1.1](#r11)
 - [v0.1.0](#v010)
 
 **Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
 
+# r1.2
+## Release Notes
+
+This release contains the definition and documentation of
+* population-density-data v0.1.1
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
```

Comment:
0.2.0-rc.2 introduced a change of the Authorization and Authentication text for the info.description, see https://github.com/camaraproject/IdentityAndConsentManagement/pull/186

Please apply this change to the API definition before the public release.

Reply:
Created https://github.com/camaraproject/PopulationDensityData/issues/46 for that.

Reply:
API is already updated.